_dashboards/admin-ui-index/datastream.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Data streams management
+parent: Index and snapshot management in OpenSearch Dashboards
+nav_order: 20
+---
+
+# Data streams management
+Introduced 2.6
+{: .label .label-purple }
+
+In OpenSearch Dashboards, the **Index Management** section allows you to view and manage [data streams]({{site.url}}{{site.baseurl}}/opensearch/data-streams/).
+
+![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams1.png)
+
+## View data streams
+
+To view data streams and their health status, choose **Data Streams** under **Index management**.
+
+The following are the three data streams health statuses:
+
+- Green: All primary and replica shards are assigned.
+- Yellow: At least one replica shard is not assigned.
+- Red: At least one primary shard is not assigned.
+
+    ![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams5.png)
+
+## Create data streams
+
+To create data streams, perform the following steps:
+
+1. Under **Index Management**, choose **Data streams**.
+
+1. Choose **Create data stream**.
+
+1. Enter a name for the data stream under **Data stream name**.
+
+2. Ensure that you have a matching index template. This will be populated under **Matching index template** as seen in the image below.
+
+    ![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams3.png)
+
+3. Under **Index alias**, select existing aliases or specify a new alias.
```

Comment:
The template detail section is readonly and used to tell users how the backing indexes of this data stream will be lilke. So after entering a name for data stream, user can only view the detail and choose create data stream.